docs/adrs/00005-license-export-analytics.md

```diff
@@ -0,0 +1,126 @@
+# 00005. Analysis of the license data structure.
+
+Date: 2025-01-23
+
+## Status
+
+DRAFT
+
+## Context
+
+#### 1 License information for the entire SBOM
+
+
+##### 1.1 CycloneDX SBOM-level License
+
+```json
+"licenses" : [
+  { 
+    "license" : { 
+      "id" : "Apache-2.0" 
+    }
+  }
+]
+```
+
+##### 1.2 SPDX SBOM-level License
+```json
+"dataLicense": "CC0-1.0",
+```
+If I understand correctly, CycloneDX's SBOM-level License refers to the license of the package corresponding to the SBOM, while SPDX's SBOM-level License refers to the license of the SBOM document itself.
```

Comment:
Lets keep the language in this ADR to assertive eg. 

CycloneDX's SBOM-level License refers to the license of the package corresponding to the SBOM, while SPDX's SBOM-level License refers to the license of the SBOM document itself

If you do not understand correctly - the ADR is not the place to question yourself ;)